app/uk/gov/hmrc/cdsreimbursementclaimfrontend/models/Eori.scala

```diff
@@ -22,5 +22,11 @@ import play.api.libs.json.Format
 final case class Eori(value: String) extends AnyVal
 
 object Eori {
+
+  def isValid(maybeEori: String): Boolean = {
+    val regex = """\w{17}"""
```

Comment:
I have asked Richard to find out what the correct format for an EORI number is.

Reply:
I did that ... it's an17 - 17 alphanemeric